packages/eslint-plugin/docs/rules/no-type-alias.md

```diff
@@ -600,6 +601,61 @@ type Foo = Partial<Bar>;
 type Foo = Omit<Bar, 'a' | 'b'>;
 ```
 
+### `allowTemplateLiterals`
```

Comment:
This should mention that the option falls back to the value for `allowAliases`. That's what I would expect.